### PR TITLE
Added documentation for the Sector Alarm integration

### PR DESCRIPTION
**Pull request by peroyvind**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27614

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/sectoralarm.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ha_category:
+  - Alarm
+  - Sensor
+ha_release: 0.101
+ha_iot_class: Cloud Polling
+---
+
+The `Sector Alarm` component adds all of your temperature sensors and a panel that lets you control your alarm from Home Assistant.
+
+This has currencly only been testet on norwegian installations, but it should work for other countries as well.
```
Comment (frenck) [inline source/_integrations/sectoralarm.markdown]:
```suggestion
This has currently only been tested on Norwegian installations, but it should work for other countries as well.
```

File: source/_integrations/sectoralarm.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Configuration
+
+Go to [minside.sectoralarm.no/](https://minside.sectoralarm.no/) and login to find your panel id. After you login it should be a part of the URL. Example: https://minside.sectoralarm.no/#!/systems/**012345678**
```
Comment (frenck) [inline source/_integrations/sectoralarm.markdown]:
```suggestion
Go to [minside.sectoralarm.no/](https://minside.sectoralarm.no/) and login to find your panel id. After you log in, it should be a part of the URL. Example: https://minside.sectoralarm.no/#!/systems/**012345678**
```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there, @peroyvind!

Thanks for the PR! Reviewed it, and left a couple of comments. Could you please take a look? Thanks! 👍 

PS: Consider adding a logo, it really helps users with identifying the brand that integrates.

**Comment by peroyvind** [comment]
> Hi there, @peroyvind!
> 
> Thanks for the PR! Reviewed it, and left a couple of comments. Could you please take a look? Thanks! 👍
> 
> PS: Consider adding a logo, it really helps users with identifying the brand that integrates.

Thanks. Fixed and logo added :)

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — APPROVED** [review]

**Pull request closed without merging**